styles/shared/comments.scss

```diff
@@ -39,6 +39,15 @@
       font-size: 12px;
       color: #FF5A5F;
     }
+
+    .p-comment-actions {
+      text-align: right;
+    }
+
+    .p-comment-cancel {
+      margin-right: 10px;
+      margin-left: 5px;
+    }
```

Comment:
`.p-*` classes (from the old Ember client) existed solely for the purpose of automated testing. We should never use them for styling.